client/api_challenge.go

```diff
@@ -29,8 +28,23 @@ type IChallengeClient interface {
 	ChallengeParams(ctx context.Context, req *challengetypes.QueryParamsRequest) (*challengetypes.QueryParamsResponse, error)
 }
 
-// GetChallengeInfo  sends request to challenge and get challenge result info
-// The challenge info includes the piece data, piece hash roots and integrity hash corresponding to the accessed SP
+// GetChallengeInfo sends request to storage provider, and returns the integrity hash and data stored on the sp.
```

Comment:
For this API, we should add more details, only validators can call this API? How SP verify the client?

Reply:
Add more information.